tools/x13as_html/makefile.gf

```diff
@@ -1,14 +1,6 @@
 # MKMF template makefile for protected mode executables.
 # FC        = gfortran
-LINKER    = $(FC)
-PROGRAM         = x13as_html
-DEST      = .
-EXTHDRS         =
-HDRS            =
-LDFLAGS   = -s
-LDMAP     =
-LIBS      =
-MAKEFILE  = Makefile
+PROGRAM         = x13ashtml
```

Comment:
I think we take Makefile.gf 'as is' from upstream so once we have this working it may be easier to keep our changes in a separate file such as, say, Makefile.package.